<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="UART">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QComboBox" name="m_listUARTs"/>
      </item>
      <item>
       <widget class="QPushButton" name="m_btnOpenUART">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>  Open</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/new/icons/Icons/LedGray.png</normaloff>:/new/icons/Icons/LedGray.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QFrame" name="verticalFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="m_btnReadRegMDIO0">
            <property name="text">
             <string>Read</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>reg 0x</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_regNrForMDIO0">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_regReadDataForMDIO0">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_btnWriteRegMDIO0">
            <property name="text">
             <string>Write</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="m_cbTx0_100F">
            <property name="text">
             <string>TX100F</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_cbTx0_100H">
            <property name="text">
             <string>TX100H</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_cbTx0_10F">
            <property name="text">
             <string>TX10F</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_cbTx0_10H">
            <property name="text">
             <string>TX10H</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_cbAutoNeg_0">
            <property name="text">
             <string>Auto Neg</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_comboSpeed_0">
            <item>
             <property name="text">
              <string>10M</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100M</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1G</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Reserved</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_cbFullDuplex_0">
            <property name="text">
             <string>Full Duplex</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="m_reset_phy_0">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_btnReNegotiate0">
            <property name="text">
             <string>Re Negotiate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="m_ledLink_0">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>16</horstretch>
              <verstretch>16</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="pixmap">
             <pixmap resource="icons.qrc">:/new/icons/Icons/LedGray.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Link</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_btnUpdateUi_0">
            <property name="text">
             <string>Read</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="verticalFrame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="m_btnReadRegMDIO1">
            <property name="text">
             <string>Read</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>reg 0x</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_regNrForMDIO1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>0x00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_regReadDataForMDIO1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_btnWriteRegMDIO1">
            <property name="text">
             <string>Write</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_comboSpeed_1">
            <item>
             <property name="text">
              <string>10M</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100M</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1G</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Reserved</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="m_listEthCardNames"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="m_btnOpenEthCard">
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnCloseEthCard">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Cllose</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnSendPkt">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Send Pkt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Loops</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_loopsCnt">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_cbSleepBetween">
         <property name="text">
          <string>Sleep between</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnExportBad">
         <property name="text">
          <string>Export Bad</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnCrewateTestPkt">
         <property name="text">
          <string>Create Test Pkt</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
